contents/tutorials/zendesk-session-replays.md

```diff
@@ -0,0 +1,226 @@
+---
+title: How to add session replays to Zendesk
+date: 2024-02-07
+author: ["ian-vanagas"]
+tags: ['session replay']
+---
+
+Session replays can be a useful support tool for debugging and recreating issues. The errors, console, and network data along with the rest of PostHog's tools make PostHog a powerful support platform.
+
+To get easy access to session replays in Zendesk, you can add a link to them when users submit them. To show you how to do this, we build a basic Next.js app with a form, add PostHog, connect the form to Zendesk, and setup the session replay link in the submitted ticket.
+
+## Creating a basic Next.js app with a form
```

Comment:
I prefer numbered headings like this. It makes the tutorial easy to follow and easier to find the information you need. I think it also increases the likelihood Google will show you in a featured snippet.

 Feel free to ignore if you prefer not to have it

```suggestion
## 1. Create a basic Next.js app with a form
```